plugins/versionpress/tests/test-helper-classes.php

```diff
@@ -2,7 +2,7 @@
 
 $directoryIterator = new RecursiveDirectoryIterator(__DIR__, FilesystemIterator::SKIP_DOTS);
 $filterIterator = new RecursiveCallbackFilterIterator($directoryIterator, function (SplFileInfo $current, $key, RecursiveDirectoryIterator $iterator) {
-    $ignoredDirectories = array('node_modules', 'vagrant');
+    $ignoredDirectories = array('node_modules');
```

Comment:
Oh, this file can be totally removed. It's a leftover from #672. I think it's not necessary to create a separate issue for that and we can do it in this PR.